p2panda-group/src/crypto/ed25519.rs

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+//! Edwards-Curve Digital Signature Algorithm (EdDSA) related to Curve25519 using SHA-512.
+//!
+//! <https://www.rfc-editor.org/rfc/rfc8032>
+use std::fmt;
+
+use curve25519_dalek::scalar::clamp_integer;
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+
+use crate::crypto::Secret;
+
+/// 256-bit signing key.
+pub const SIGNING_KEY_SIZE: usize = 32;
```

Comment:
Another naming nitpick:

In `aead.rs` we use `_LEN` suffix and here we use `_SIZE`. Maybe stick with one throughout the crate?

Reply:
The `_LEN` suffix comes from an external crate which is not further exported by us, so I guess it's fine.